src/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -343,7 +344,7 @@ public static Delegate CreateDelegate(Type type, object target, string method, b
             if (method == null)
                 throw new ArgumentNullException(nameof(method));
 
-            RuntimeType rtType = type as RuntimeType;
+            RuntimeType? rtType = type as RuntimeType;
```

Comment:
We could write this as
```cs
if (!(type is RuntimeType rtType))
```

Reply:
Hmm, but it seems like `rtType` really is nullable, since on the very next line we check it against null. And the compiler even figures that out, and doesn't complain on the line after that when I dereference it. So if I understand correctly, what I have here is correct.